packages/contracts/package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "@zk-email/contracts",
-  "version": "5.0.2",
+  "version": "6.0.0",
```

Comment:
Let me use 6.0.0 in ether-email-auth.

Reply:
Hmmm I'm not sure it matters much since they will get out of sync anyways